src/ui/Deck.js

```diff
@@ -0,0 +1,83 @@
+export class DeckNotFoundError extends Error {
+    constructor(message) {
+        super(message);
+        this.name = "DeckNotFoundError";
+    }
+}
+
+export class Deck {
+    deckId;
+    expiry;
+
+    async init(id, expiry) {
```

Comment:
@semmypurewal this was a holdover from our API since `/deck/new` actually deals a 5-card hand. In hindsight we should've separated those API calls out and let `deal` take care of it.

Reply:
@semmypurewal I actually thought of that, but our API made it tough since it returns both a deck ID (and expiration) and deals us a 5-card hand. In hindsight, that was a poor design decision. Supposing we keep the API the same, I'm not sure how this should be dealt with on the client.